extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/FinalDestinationBuildItem.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.smallrye.graphql.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class FinalDestinationBuildItem extends SimpleBuildItem {
```

Comment:
Is this class meant to be used outside the extension? Otherwise it would be nice to be marked as package-private to avoid being listed in https://quarkus.io/guides/all-builditems

Reply:
Otherwise it would be nice to add some Javadoc to this class :+1:

Reply:
Nope this is for internal only, I'll change to package-private. Thanks :)